scripts/test_execution.json

```
@@ -27,6 +27,78 @@
                 "Name": "vm2"
             }
         ],
+        "TK5-3WP07R0703_2":
```

```suggestion
        "TK5-3WP07R0703_WS2019_2":
```
for consistency with line 54.
Same throughout.

---

Because there's no way to rename existing runners, we will need a short branch lockdown for me to uninstall and re-install all of them with new names.